### PR TITLE
Add The Feature form to the project 

### PR DESCRIPTION
**Pull request by Mujeeb4582**
Add the background image of the form
Add the form in index.html
Add also the styling for form in the style.css
with the help of @Oscarj-8. we are code partners.
 
I think there is a version difference between my vs code editor and @Oscarj-8 vs code editor. 
When he clones my repository in his local computer, his vs code automatically makes a lot of changes like spacing, and line numbering and converts the single quotation mark to a double quotation mark.
Dear sir, read his commit description, and according to that observe the changes in index.html and style.css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,293 +1,354 @@
...
+        </div>
+      </section>
+
+			      <!-- Skills end -->
+
+      <!-- About section end -->
+
+      <!-- Form start  -->
+
+      <form>
```
Comment (topeogunleye) [inline index.html]: - [ ] As required by project required. You're required to use  formspree for for submittion and therefore you're supposed to have on your form:

- An `action` value similar to `https://formspree.io/f/{form_id}`
- A `method` value equals `POST`

File: index.html [hunk 2]
```diff
@@ -1,293 +1,354 @@
...
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="email_address"
+            placeholder="Email address"
+            required
+          />
+          <textarea name="message" id="message" maxlength="300">
```
Comment (topeogunleye) [inline index.html]: - [ ] required attribute is needed here so as to prevent users submitting an empty form

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Mujeeb4582, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your @topeogunleye so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Mujeeb4582) [reply index.html]: @topeogunleye required attribute in textarea tag???

**Comment by Mujeeb4582** [comment]
@topeogunleye  
in general comments what is the meaning of 
x, y and z.

**Comment by topeogunleye** [comment]
> @topeogunleye in general comments what is the meaning of x, y and z.

 I forgot to remove it.

File: index.html (hunk 2 above)
Comment (Mujeeb4582) [reply index.html]: @topeogunleye what about this?? 
required attribute in textarea tag?

Comment (topeogunleye) [reply index.html]: @Mujeeb4582, Yes pls.

File: index.html [hunk 3]
```diff
@@ -285,7 +285,7 @@ <h3 class="Skills-title">Skills</h3>
 
       <!-- Form start  -->
 
-      <form>
+      <form action="https://formspree.io/f/{form_id}" method="post">
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ optional ] Good job for using form-spree for for submission, But when you are using [formspree service](https://formspree.io/html) you have to create your own hash id  <form action="https://formspree.io/f/{your_id}" method="post">. When I submitted it show the below image,
Capture

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Mujeeb4582      , 

Your project is complete .  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional  README
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Mujeeb4582) [reply index.html]: check the bellow commit

Comment (Abenezer-Tilahun) [reply index.html]: Hello @Mujeeb4582 Good Job 👍

Comment (Mujeeb4582) [reply index.html]: check the latest commit

Comment (Mujeeb4582) [reply index.html]: @Abenezer-Tilahun

Comment (Abenezer-Tilahun) [reply index.html]: I checked it and it done well , I approved the Project and go head and merge it.

Comment (Mujeeb4582) [reply index.html]: thank you sir

**Pull request merged**